core/server/middleware/theme-handler.js

```diff
@@ -60,6 +60,8 @@ themeHandler = {
 
         // clear the view cache
         blogApp.cache = {};
+        // reset the asset hash
+        config.assetHash = null;
```

Comment:
oh yes true, overrides!
we could store a hash of theme name and updated_at into express app, so the whole theme activation logic in middleware layer get's triggered.